### DISCUSSION TITLE
Codemagic doesn't accept my iOS testing group?

### DISCUSSION THREAD

**Discussion by Gauerdia** [codemagic-ci-cd/codemagic-docs#2509] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Hello, I am trying to build and distribute a flutter app via codemagic to the apple store.
1. The build in codemagic works fine, so that's not the issue
2. I created a test group with a test phone in the "TestFlight" part of the app in the app store connect
3. But everytime I start my codemagic build, I get the following error:
"Cannot create profile: the request does not include any iOS testing devices while they are required for creating a IOS_APP_ADHOC profile. If the profile creation is automatic, ensure that at least one suitable testing device is registered on the Apple Developer Portal."

What do I need to do?

### Expected results

Everything goes through and my test device is asked for testing.

### Actual results

error in codemagic.

**Comment by kterto** [comment]
I think the problem is with your sign in configuration with apple... could you review the configuration? 
https://docs.codemagic.io/flutter-code-signing/ios-code-signing/
and pay special attention to the requirement: "Note that an Ad Hoc or Enterprise distribution profile is required to distribute the app outside your development team."

**Comment by dtrdic** [comment]
Hey @Gauerdia, when generating a Provisioning Profile on the Apple developer page have you selected the devices you wish to include in this provisioning profile? Please also share a build id so i can take a look at the logs.

But also, if this device is not registered in your developer account, you will need to do this first.

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
